driver-bindings/src/bindings.rs

```diff
@@ -1,4 +1,4 @@
-/* automatically generated by rust-bindgen 0.59.2 */
+/* automatically generated by rust-bindgen 0.62.0 */
```

Comment:
Is [this](https://github.com/aws/aws-nitro-enclaves-cli/blob/main/Cargo.toml#L35) used anywhere? If not, we should probably remove it.

Reply:
It doesn't seem to be used anywhere.

Reply:
I'll check if it can be removed in a future PR.

Reply:
Ok, thanks!